### PR TITLE
Install Postgresql server RPMs to prevent selinux issue

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: katello/hooks/pre/32-install_postgresql.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Working around https://tickets.puppetlabs.com/browse/PUP-2169
+if local_postgresql? && facts[:os][:release][:major] == '7' && !app_value(:upgrade)
+  `rpm -qa | grep rh-postgresql10-postgresql-server-syspaths`
+  unless $?.success?
+    Kafo::Helpers.execute('yum -y install rh-postgresql10-postgresql-server-syspaths')
```
Comment (evgeni) [inline katello/hooks/pre/32-install_postgresql.rb]: I know this is not the `upgrade` path, so the system *should* be clean, but would it be sufficient to install `rh-postgresql10-postgresql-server` and let the installer install `rh-postgresql10-postgresql-server-syspaths` later?

File: katello/hooks/pre/32-install_postgresql.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Working around https://tickets.puppetlabs.com/browse/PUP-2169
+if local_postgresql? && facts[:os][:release][:major] == '7' && !app_value(:upgrade)
+  `rpm -qa | grep rh-postgresql10-postgresql-server-syspaths`
```
Comment (evgeni) [inline katello/hooks/pre/32-install_postgresql.rb]:
```suggestion
  `rpm -q rh-postgresql10-postgresql-server-syspaths`
```

no need to `grep`, `rpm` will exit non-zero if the package is not installed.

File: katello/hooks/pre/32-install_postgresql.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Working around https://tickets.puppetlabs.com/browse/PUP-2169
+if local_postgresql? && facts[:os][:release][:major] == '7' && !app_value(:upgrade)
+  `rpm -q rh-postgresql10-postgresql-server`
```
Comment (ekohl) [inline katello/hooks/pre/32-install_postgresql.rb]: I'd prefer the package name in a variable

**Review by ekohl — APPROVED** [review]
I'd prefer this hook is number 29 and drops the `!app_value(:upgrade)` part so it always installs it. Then you can drop it from `katello/hooks/pre/30-upgrade.rb`. Not strictly a blocker

**Review by ekohl — APPROVED** [review]

File: katello/hooks/pre/29-install_postgresql.rb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# Working around https://tickets.puppetlabs.com/browse/PUP-2169
+if local_postgresql? && facts[:os][:release][:major] == '7' && !app_value(:upgrade)
+  package = 'rh-postgresql10-postgresql-server'
```
Comment (ehelms) [inline katello/hooks/pre/29-install_postgresql.rb:3]: Added

File: katello/hooks/pre/29-install_postgresql.rb [hunk 5]
```diff
@@ -0,0 +1,10 @@
+# Working around https://tickets.puppetlabs.com/browse/PUP-2169
+if local_postgresql? && facts[:os][:release][:major] == '7' && !app_value(:upgrade)
```
Comment (ehelms) [inline katello/hooks/pre/29-install_postgresql.rb:2]: For now, I don't want this messing with the upgrade workflow given thats a very orchestrated change.

**Pull request merged**